lib/data_distributor/cpr/client.rb

```diff
@@ -0,0 +1,34 @@
+require "faraday"
+require "json"
+require "date"
+
+module DataDistributor
+  module CPR
+    # This class is used for retrieving data from the Danish CPR register
+    class Client
+      attr_reader :connection
+
+
+      def initialize(authentication=DataDistributor::Authentication::NoAuthentication.new)
+        @connection = Faraday.new(url: "https://services.datafordeler.dk/CPR/CprPrivatePNR/2.0.0/rest/") do |connection|
+          connection.use Faraday::Response::RaiseError
+          authentication.apply(connection)
+          connection.adapter Faraday.default_adapter
+        end
+      end
```

Comment:
Vi kunne overveje om vi skulle lave en `DataDistributor::Client` klasse, som indeholder denne opsætning af Faraday, da den stort set er ens for både `CPR::Client` og `BBR:Client`?

Reply:
Den er nu lavet.